packages/js/src/Modules/Verto/index.ts

```diff
@@ -20,7 +20,69 @@ export default class Verto extends BrowserSession {
     return isValidOptions(this.options);
   }
 
+  /**
+   * Makes a new outbound call.
+   *
+   * This method receives an object `options` with the following properties:
+   *
+   * @param options Options object for a new call.
+   * @param options.destinationNumber Extension to dial.
+   * @param options.callerNumber Number to use as the caller ID when dialling out to a phone number.
+   * @param options.callerName Name to use as the caller ID name when dialling out to a phone number.
+   * @param options.id The identifier of the Call.
+   * @param options.localStream If sets, the Call will use this stream instead of retrieving a new one.
+   * @param options.localElement Overrides client's default `localElement`.
+   * @param options.remoteElement Overrides client's default `remoteElement`.
+   * @param options.iceServers Overrides client's default `iceServers`.
+   * @param options.audio Overrides client's default audio constraints.
+   * @param options.video Overrides client's default video constraints.
+   * @param options.useStereo Uses stereo audio instead of mono.
+   * @param options.micId `deviceId` to use as microphone. Overrides the client's default one.
+   * @param options.camId `deviceId` to use as webcam. Overrides the client's default one.
+   * @param options.speakerId `deviceId` to use as speaker. Overrides the client's default one.
+   * @param options.onNotification Overrides client's default `telnyx.notification` handler for this Call.
+   *
+   * @return `Promise<Call>` A promise fulfilled with the new outbound Call object
```

Comment:
I would standardize around lowercase "call" when referring to the concept of a call, and `Call` in code when referring to the JS class/object:
```suggestion
   * @param options.onNotification Overrides client's default `telnyx.notification` handler for this call.
   *
   * @return `Promise<Call>` A promise fulfilled with the new outbound `Call` object
```